CONTRIBUTING.md

```diff
@@ -13,14 +13,98 @@ Check out the [Projects](https://github.com/jMonkeyEngine/jmonkeyengine/projects
 
 When you're ready to submit your code, just make a [pull request](https://help.github.com/articles/using-pull-requests).
 
-- Do not commit your code until you have received proper feedback.
 - In your commit log message, please refer back to the originating forum thread (example) for a ‘full circle’ reference. Also please [reference related issues](https://help.github.com/articles/closing-issues-via-commit-messages) by typing the issue hashtag.
 - When committing, always be sure to run an update before you commit. If there is a conflict between the latest revision and your patch after the update, then it is your responsibility to track down the update that caused the conflict and determine the issue (and fix it). In the case where the breaking commit has no thread linked (and one cannot be found in the forum), then the contributor should contact an administrator and wait for feedback before committing.
 - If your code is committed and it introduces new functionality, please edit the wiki accordingly. We can easily roll back to previous revisions, so just do your best; point us to it and we’ll see if it sticks!
 
 p.s. We will try hold ourselves to a [certain standard](http://www.defmacro.org/2013/04/03/issue-etiquette.html) when it comes to GitHub etiquette. If at any point we fail to uphold this standard, let us know.
 
-#### Core Contributors
+There are many ways
+to submit a pull request (PR) to the "jmonkeyengine" project repository,
+depending on your knowledge of Git and which tools you prefer.
+
+<details>
+    <summary>
+        <b>Click to view step-by-step instructions for a reusable setup
+        using a web browser and a command-line tool such as Bash.</b>
+    </summary>
+
+The setup described here allows you to reuse the same local repo for many PRs.
+
+#### Prerequisites
+
+These steps need only be done once...
+
+1. Create a personal account at GitHub, if you don't already have one:
+  + Browse to https://github.com/
+  + Click on the "Sign up" button in the upper right.
+  + Follow the instructions.
+2. Sign into your GitHub account, if you're not already signed in:
+  + Browse to https://github.com/
+  + Click on the "Sign in" button in the upper right.
+  + Follow the instructions.
```

Comment:
At what point do we have to assume that someone who has the skills to make edits to computer source code also has the ability to sign up and sign into a website?

I might distill these entire two points down to:
"You will need a personal account on https://github.com/ . Signup and Sign-in are in the upper right corner."

I would be glad of feedback from others who might have a different perspective/experience, though.

Reply:
I'll compress this as you suggest.